Dalamud/Game/Gui/ChatGui.cs

```diff
@@ -493,6 +502,46 @@ private void HandleLinkClickDetour(LogViewer* thisPtr, LinkData* linkData)
             Log.Error(ex, "Exception in HandleLinkClickDetour");
         }
     }
+
+    private void UpdateDetour(RaptureLogModule* thisPtr)
+    {
+        try
+        {
+            foreach (ref var item in thisPtr->LogMessageQueue)
+            {
+                var logMessage = new Chat.LogMessage((LogMessageQueueItem*)Unsafe.AsPointer(ref item));
+
+                // skip any entries that survived the previous Update call as the event was already called for them
+                if (this.seenLogMessageObjects.Contains(logMessage.Address))
+                    continue;
```

Comment:
Are there any cases where the game normally sends 2 of the same log messages in a row that would be valid to trigger on?